src/groups/mqb/mqbc/mqbc_storagemanager.h

```diff
@@ -768,6 +806,11 @@ class StorageManager
     /// THREAD: Executed by the cluster's dispatcher thread.
     virtual void stop() BSLS_KEYWORD_OVERRIDE;
 
+    /// Initialize the queue key info map based on information in the specified
+    /// `clusterState`.
+    virtual void initializeQueueKeyInfoMap(
+        const mqbc::ClusterState* clusterState) BSLS_KEYWORD_OVERRIDE;
```

Comment:
`clusterState` is a const parameter, it's not optional and required every time. Is it possible to convert it to const reference?

Reply:
Agreed

Reply:
d